dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -1850,4 +1850,59 @@ public void testCreateItemInvalidCollectionBadRequestException() throws Exceptio
                         .andExpect(status().isBadRequest());
     }
 
+    @Test
+    public void updateTestEPersonWithoutPermissionForbidden() throws Exception {
+        //We turn off the authorization system in order to create the structure as defined below
+        context.turnOffAuthorisationSystem();
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        ObjectMapper mapper = new ObjectMapper();
+        ItemRest itemRest = new ItemRest();
+        itemRest.setName("Practices of research data curation in institutional repositories:" +
+                             " A qualitative view from repository staff");
+        itemRest.setInArchive(true);
+        itemRest.setDiscoverable(true);
+        itemRest.setWithdrawn(false);
+
+
+        String token = getAuthToken(admin.getEmail(), password);
+        MvcResult mvcResult = getClient(token).perform(post("/api/core/items?owningCollection=" +
+                                                                col1.getID().toString())
+                                                           .content(mapper.writeValueAsBytes(itemRest))
+                                                           .contentType(contentType))
+                                              .andExpect(status().isCreated())
+                                              .andReturn();
```

Comment:
This first test appears to be the *opposite* of what the method suggests it's testing.  The method is named `updateTestEPersonWithoutPermissionForbidden()`,  but this is testing that an Admin is not Forbidden.  Maybe this should be a separate method?

Reply:
The flow of the test is designed in such a way so that we create an item through a REST call first and we'll use that item to test the update instead of picking any random item that had been used throughout the test class or made through one of the builders. This way we can ensure that the item that's actually created cannot be malformed due to our tests in any way shape or form and that it's an accurate representation of how it'd be as if it were actually created by a user (admin in this case) in a deployed environment. This is the item that gets created by the admin and checks that it's created properly. We'll then retrieve the UUID for this item and the second call will try to update this item with an EPerson that doesn't have permission, which should throw the Forbidden exception.
I made this test to be very similar to the `updateTest()` a bit higher up in the test class, the only difference is that the token gets changed from an admin to a normal eperson to test the permission